src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1898,6 +1514,11 @@ CreateSubscriptions(MultiConnection *sourceConnection,
 					GetUserNameFromId(ownerId, false)
 					)));
 
+		InsertCleanupRecordInSubtransaction(CLEANUP_OBJECT_USER,
+											target->subscriptionOwnerName,
+											worker->groupId,
+											policy);
```

Comment:
nit: I think I prefer to replace this simply with CLEANUP_ALWAYS instead of having an intermediary variable. It makes the code easier to read IMHO
```suggestion
											CLEANUP_ALWAYS);
```